README.md

```diff
@@ -171,6 +171,16 @@ Some things to keep in mind if you create your own instance of TurbolinksSession
 
 You'll need to weigh the benefits and complexities of those options, but the bottom line is that you'll want to carefully manage the lifecycle of your Turbolinks instance(s).
 
+### Custom TurbolinksSession Settings
```

Comment:
This might be better named "Overriding Default TurbolinksSession Settings" to help indicate this is a section on 1) what the default settings are and 2) how to override them.